### PR TITLE
Calendar bug

### LINKED ISSUE #2349: Crash: App crashes if default calendar app is disabled
**Actual Behaviour**

screenshot_20180218-180318

**Expected Behaviour**

Display a toast that calendar app is unavailable

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by sriramr98**
Fixes #2349 
Changes: Added a check to make sure a calendar app is present before navigating to the calendar.

### CODE REVIEW

**Comment by sriramr98** [comment]
@iamareebjamal can you review this PR?

File: android/app/src/main/java/org/fossasia/openevent/core/track/session/SessionDetailActivity.java [hunk 1]
```diff
@@ -413,7 +414,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 intent.putExtra(CalendarContract.Events.DESCRIPTION, Html.fromHtml(session.getShortAbstract()).toString());
                 intent.putExtra(CalendarContract.EXTRA_EVENT_BEGIN_TIME, DateConverter.formatDateWithDefault(DateConverter.FORMAT_24H, session.getStartsAt()));
                 intent.putExtra(CalendarContract.EXTRA_EVENT_END_TIME, DateConverter.formatDateWithDefault(DateConverter.FORMAT_24H, session.getEndsAt()));
-                startActivity(intent);
+                if (intent.resolveActivity(getPackageManager()) != null){
+                    startActivity(intent);
+                }else{
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/track/session/SessionDetailActivity.java]: Spaces around else

**Comment by sriramr98** [comment]
@iamareebjamal done.

**Review by iamareebjamal — APPROVED** [review]

File: android/app/src/main/java/org/fossasia/openevent/core/track/session/SessionDetailActivity.java [hunk 2]
```diff
@@ -413,7 +414,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 intent.putExtra(CalendarContract.Events.DESCRIPTION, Html.fromHtml(session.getShortAbstract()).toString());
                 intent.putExtra(CalendarContract.EXTRA_EVENT_BEGIN_TIME, DateConverter.formatDateWithDefault(DateConverter.FORMAT_24H, session.getStartsAt()));
                 intent.putExtra(CalendarContract.EXTRA_EVENT_END_TIME, DateConverter.formatDateWithDefault(DateConverter.FORMAT_24H, session.getEndsAt()));
-                startActivity(intent);
+                if (intent.resolveActivity(getPackageManager()) != null){
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/track/session/SessionDetailActivity.java]: Space after `)`

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/org/fossasia/openevent/core/track/session/SessionDetailActivity.java (hunk 2 above)
Comment (sriramr98) [reply android/app/src/main/java/org/fossasia/openevent/core/track/session/SessionDetailActivity.java]: which ) do you want the change? The ones in the if statement?

**Comment by sriramr98** [comment]
@iamareebjamal done

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**